com.unity.cinemachine/Runtime/Core/InputAxis.cs

```diff
@@ -211,10 +212,10 @@ struct RecenteringState
         RecenteringState m_RecenteringState;
 
         /// <summary>
-        /// Call this before calling DoRecentering.  Will track any vlue changes so that the recentering clock
+        /// Call this before calling DoRe-centering.  Will track any value changes so that the re-centering clock
```

Comment:
DoRecentering is a method name, it should be left alone

Reply:
Fixed